docs/articles/documentation/reference/test-api/test/httpauth.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Test.httpAuth Method
+permalink: /documentation/reference/test-api/test/httpauth.html
+---
+# Test.httpAuth Method
+
+Specifies the credentials for HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) and [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication.
+
+```text
+test.httpAuth( credentials )
+```
+
+{% include authentication/http-credentials.md %}
+
+TestCafe uses the specified credentials for all requests that require authentication in the fixture. To specify credentials to be used in fixtures, use the [fixture.httpAuth](../fixture/httpauth.md) method.
```

Comment:
These two sentences are very unclear when close together. The second one definitely needs more details.

Reply:
```suggestion
TestCafe uses the specified credentials for all requests that require authentication in the individual test. To specify credentials to be used in fixtures, use the [fixture.httpAuth](../fixture/httpauth.md) method.
```